docker-compose.yml

```diff
@@ -50,10 +50,6 @@ services:
       ODE_RSU_USERNAME: ${ODE_RSU_USERNAME}
       ODE_RSU_PASSWORD: ${ODE_RSU_PASSWORD}
       DEFAULT_SNMP_PROTOCOL: ${DEFAULT_SNMP_PROTOCOL}
-      # Commented out, will use SDW depositor module by default
-      #ODE_DEPOSIT_SDW_MESSAGES_OVER_WEBSOCKET: ${ODE_DEPOSIT_SDW_MESSAGES_OVER_WEBSOCKET}
```

Comment:
Are there any code remnants of these old variables that need to be removed?

Reply:
Yes! The modifications involved to remove them seemed outside the scope of this work item and better suited to a separate work item. I'll make a note to open a work item for this.